### PR TITLE
Added maths expression alt text example

### PR DESCRIPTION
**Pull request by cmuncharaz**
Added an example explaining how to write alt text for images of mathematical expressions.

### CODE REVIEW

File: accessibility/alt-text.md [hunk 1]
```diff
@@ -196,3 +196,20 @@ This alt text does the following things:
...
+In the case of images of mathematical expressions, the alt text should describe the expression in spoken mathematics. Avoid using special characters matching mathematical symbols, such as operators, fractions, etc because they might not be announced as expected by screen readers.
+
+#### Insufficient alt text
+
+* "(a + b) / c"
+
+#### Sufficient alt text
+
+* "The quantity of a plus b divided by c."
+* "Fraction where the numerator is a plus b and the denominator is c."
```
Comment (ruthmoog) [inline accessibility/alt-text.md:215]:
```suggestion
* "Fraction where the numerator is a plus b and the denominator is d."
```

File: accessibility/alt-text.md [hunk 2]
```diff
@@ -196,3 +196,20 @@ This alt text does the following things:
 1. Incorporates, verbatim, the text to the left of the cover image
 2. Details the title of the book, the name of the book’s author, the publishing house, and the publication series the book belongs to - all visible on the cover image
 3. Describes the cover illustration; which is of a highly-recognisable film character that’s directly relevant to the book’s subject matter, and that the target audience for this book would be expected to be familiar with
+
+### Mathematical Expressions
+
+![Screenshot of a faction where the numerator is a plus b and the denominator is c](images/alt-examples/maths.png)
```
Comment (ruthmoog) [inline accessibility/alt-text.md:202]:
```suggestion
![Fraction where the numerator is a plus b and the denominator is d](images/alt-examples/maths.png)
```

File: accessibility/alt-text.md [hunk 3]
```diff
@@ -196,3 +196,20 @@ This alt text does the following things:
...
+
+The preferred method to convey mathematical expressions should be using [MathML](https://www.w3.org/Math/) (a standard markup language for describing mathematical notation) or [MathJax](https://www.mathjax.org/) (a JavaScript display engine for mathematics). However, images of mathematical expressions have been used traditionally as a way to easily implement them. It’s worth sharing some examples of good alt text for those cases.
+
+In the case of images of mathematical expressions, the alt text should describe the expression in spoken mathematics. Avoid using special characters matching mathematical symbols, such as operators, fractions, etc because they might not be announced as expected by screen readers.
```
Comment (ruthmoog) [inline accessibility/alt-text.md:206]:
```suggestion
In the case of images of mathematical expressions, the alt text should describe the expression in spoken mathematics. Avoid using symbols or characters with mathematical usages (such as operators, fractions, units etc) which may not be announced as expected by screen readers.
```

**Review by ruthmoog — CHANGES_REQUESTED** [review]
Really cool addition, thanks!

- suggestions for readability and fix examples so they match the image

**Review by RSeidelsohn — APPROVED** [review]
This looks like a great addition!

File: accessibility/alt-text.md [hunk 4]
```diff
@@ -196,3 +196,20 @@ This alt text does the following things:
...
+
+![Fraction where the numerator is a plus b and the denominator is d](images/alt-examples/maths.png)
+
+The preferred method to convey mathematical expressions should be using [MathML](https://www.w3.org/Math/) (a standard markup language for describing mathematical notation) or [MathJax](https://www.mathjax.org/) (a JavaScript display engine for mathematics). However, images of mathematical expressions have been used traditionally as a way to easily implement them. It’s worth sharing some examples of good alt text for those cases.
+
+In the case of images of mathematical expressions, the alt text should describe the expression in spoken mathematics. Avoid using symbols or characters with mathematical usages (such as operators, fractions, units etc) which may not be announced as expected by screen readers.
+
+#### Insufficient alt text
+
+* "(a + b) / c"
```
Comment (ruthmoog) [inline accessibility/alt-text.md:210]:
```suggestion
* "(a + b) / d"
```

File: accessibility/alt-text.md [hunk 5]
```diff
@@ -196,3 +196,20 @@ This alt text does the following things:
...
+
+#### Sufficient alt text
+
+* "The quantity of a plus b divided by c."
```
Comment (ruthmoog) [inline accessibility/alt-text.md:214]:
```suggestion
* "The quantity of a plus b divided by d."
```
Sorry didn't see these ones earlier! having all the same helps with clarity

**Review by ruthmoog — CHANGES_REQUESTED** [review]

**Review by ruthmoog — APPROVED** [review]

**Pull request merged**